Dockerfile

```diff
@@ -25,6 +25,21 @@ COPY Makefile Makefile
 COPY database/sqlite database/sqlite
 RUN make sqlite-db-refresh
 
+# INSTALL POSTGRESQL
```

Comment:
Keep the existing Dockerfile as is. I would move the PostgreSQL-related changes into a new Dockerfile named `Dockerfile-Postgres` so that we don't mix up the databases and also keep the existing docker image light-weight.